Frontend/android-native/app/src/main/java/in/juspay/mobility/utils/MyFirebaseMessagingService.java

```diff
@@ -375,7 +378,8 @@ public void restartLocationService()
     private void startWidgetService(String widgetMessage, JSONObject data, JSONObject payload){
         SharedPreferences sharedPref = this.getSharedPreferences(this.getString(R.string.preference_file_key), Context.MODE_PRIVATE);
         Intent widgetService = new Intent(getApplicationContext(), WidgetService.class);
-        if (getResources().getString(R.string.service).equals(getString(R.string.nammayatripartner)) && Settings.canDrawOverlays(getApplicationContext())  && !sharedPref.getString(getResources().getString(R.string.REGISTERATION_TOKEN), "null").equals("null") && (sharedPref.getString(getResources().getString(R.string.ACTIVITY_STATUS), "null").equals("onPause") || sharedPref.getString(getResources().getString(R.string.ACTIVITY_STATUS), "null").equals("onDestroy"))) {
+        String merchantType = BuildConfig.MERCHANT_TYPE;
```

Comment:
Can we define this `merchantType` globally, instead of defining it multiple times